host/2.0/alpine/amd64/dotnet/dotnet.Dockerfile

```diff
@@ -15,10 +15,10 @@ RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
 
 RUN apt-get update && \
     apt-get install -y gnupg wget unzip && \
-    wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.1.1/Microsoft.Azure.Functions.ExtensionBundle.1.1.1.zip && \
-    mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.1.1 && \
-    unzip /Microsoft.Azure.Functions.ExtensionBundle.1.1.1.zip -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.1.1 && \
-    rm -f /Microsoft.Azure.Functions.ExtensionBundle.1.1.1.zi
+    wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/1.3.0/Microsoft.Azure.Functions.ExtensionBundle.1.3.0.zip && \
```

Comment:
This will not work since bundle is not present on the cdn yet. We upload the bundle to the cdn last. Also the zip file name has changed since the new bundle includes readyToRunBinaries

Reply:
Yea, I know, should've change the title to [WIP]. This PR is just to remind us update the extension bundle in the /build release.

Reply:
You can change the zip file name to `Microsoft.Azure.Functions.ExtensionBundle.1.3.0_linux-x64.zip` it is live on cdn now. Have only uploaded the zip. So the tooling won't be affected. Core tools and VS Code use index.json file to check if a new bundle is available. I will update the index.json once Rapid update and the linux roll out is complete.